conf/repo/maintainer.yml

```diff
@@ -1,12 +1,14 @@
 maintainer-quality-tools:
-  branches: []
+  branches:
+    - "18.0"
```

Comment:
BTW, maintainer-quality-tools is archived, so we can remove it from here

Reply:
I'll do a separate PR for this.

Reply:
[Shall we archive https://github.com/OCA/runbot-addons too?](https://github.com/OCA/repo-maintainer-conf/pull/37)